src/miniscript/satisfy.rs

```diff
@@ -58,7 +59,8 @@ pub trait Satisfier<Pk: MiniscriptKey + ToPublicKey> {
     /// Given a raw `Pkh`, lookup corresponding [`bitcoin::secp256k1::XOnlyPublicKey`]
```

Comment:
I cannot annotate above but in L40 there's a "look" -> "looks" and "schnorr" -> "Schnorr"